src/components/SEO.tsx

```diff
@@ -0,0 +1,59 @@
+import { NextSeo } from 'next-seo';
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import { title } from 'process';
+
+import {
+  SEO_TITLE,
+  SEO_DESCRIPTION,
+  SEO_KEYWORDS,
+  SEO_OG_IMAGE,
+  SEO_TWITTER_IMAGE,
+} from '../constants';
+
+export type SEOProps = {
+  title?: string;
+  description?: string;
+  keywords?: string;
+  ogTitle?: string;
+  ogDescription?: string;
+  ogImage?: string;
+  ogUrl?: string;
+  twitterTitle?: string;
+  twitterDescription?: string;
+  twitterImage?: string;
+};
+
+export const SEO = (props: SEOProps) => {
```

Comment:
I would destructure the props here, and assign them a default value. So we can rid if all the `??`

Reply:
Yeah that sounds good